bundles/automation/org.eclipse.smarthome.automation.integration.test/org.eclipse.smarthome.automation.integration.test.launch

```diff
@@ -26,7 +26,7 @@
 <stringAttribute key="org.eclipse.jdt.junit.TESTNAME" value=""/>
 <stringAttribute key="org.eclipse.jdt.junit.TEST_KIND" value="org.eclipse.jdt.junit.loader.junit4"/>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
Actually the automation sources are 1.7 compatible because the ESH was 1.7. When the ESH goes to jdk 1.8 we have decided with Kai that they can be 1.8 compact 2 (compact 2 because the GSON requires compact2). We are trying to be close to 1.7 as match as possible to have opportunity to support jvm such as skelmir. 
I've changed required jvm because the test can be started. It depends from bundles which required 1,8 jvm and there are a lot of such exceptions: 

```
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.smarthome.automation.providers [5]
  Unresolved requirement: Import-Package: org.eclipse.smarthome.automation
    -> Export-Package: org.eclipse.smarthome.automation; bundle-version="0.9.0.qualifier"; bundle-symbolic-name="org.eclipse.smarthome.automation.api"; version="0.0.0"
       org.eclipse.smarthome.automation.api [48]
         Unresolved requirement: Import-Package: org.eclipse.smarthome.config.core
           -> Export-Package: org.eclipse.smarthome.config.core; bundle-version="0.9.0.qualifier"; bundle-symbolic-name="org.eclipse.smarthome.config.core"; version="0.0.0"
              org.eclipse.smarthome.config.core [3]
                Unresolved requirement: Import-Package: org.eclipse.smarthome.core.common
                  -> Export-Package: org.eclipse.smarthome.core.common; bundle-version="0.9.0.qualifier"; bundle-symbolic-name="org.eclipse.smarthome.core"; version="0.0.0"
                     org.eclipse.smarthome.core [8]
                       Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=1.8))"

	at org.eclipse.osgi.container.Module.start(Module.java:434)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1582)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1562)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1533)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1476)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)